### ISSUE TITLE
Make `ember-template-imports` / `ember-cli-htmlbars` an optional peer dependency

### ISSUE THREAD

**Issue by gossi** [gossi/frontend-configs#60]
... for all those packages that are no ember packages. Here is a pnpm install log:

```
incubator/aria-navigator
└─┬ @gossi/config-eslint 0.2.0
  └─┬ eslint-plugin-ember 11.2.0
    └─┬ ember-template-imports 3.3.1
      └── ✕ missing peer ember-cli-htmlbars@^6.0.0
Peer dependencies that should be installed:
  ember-cli-htmlbars@^6.0.0
```

That shouldn't be the case.

**Comment by gossi** [comment]
[`ember-cli-htmlbars` has been removed as peer dependency from `ember-template-imports`](https://github.com/ember-template-imports/ember-template-imports/commit/d1c4bd3928d44ad330829202158ca66c7f12b7f0). 

This will be release as of the version after `v.3.4.0` - The resolution for this ticket is... to wait until release are made, propagated and bots take care of it 😊

-> Keep this one open until this process finished.

**Comment by gossi** [comment]
Status:

- `ember-template-imports@v3.4` is release ✅ 
- `eslint-plugin-ember` needs to bump to the version above and drop a release ❌

**Comment by gossi** [comment]
This is released with `v11.4.6` - see PR: ember-cli/eslint-plugin-ember#1767
